errors.ts

```diff
@@ -99,6 +83,26 @@ export function installUncaughtExceptionListener(actionOnException?: () => void)
 	});
 }
 
+function tryTrackException(error: Error, injector: IInjector): void {
+	let disableAnalytics: boolean;
+	try {
```

Comment:
it means that every single unit test must mock staticConfig and set disableAnalytics to true. As in the tests we are using testInjector, we should register the staticConfig in each of them. That's what I'm trying to prevent, but if you insist, I'll handle this in a separate PR :)